README.md

```diff
@@ -1,6 +1,6 @@
 # Aware
 
-Aware is a menubar app for OSX and macOS that displays how long you've been actively using your computer.
+Aware is a menubar app for OSX, macOS, and visionOS that displays how long you've been actively using your computer.
```

Comment:
Ah remember the OS X days. Maybe I need to find and replace that in some other spots. I bumped the version requirements recently (phasing out any true OS X version) to add "Open at Login" support.

```suggestion
Aware is a menubar app for macOS and visionOS that displays how long you've been actively using your computer.
```